### PR TITLE
write RPAf design decision file

### LINKED ISSUE #88: DOCUMENTING: Design decision RPA framework
As a development team, we decided to use the [RPA framework](https://rpaframework.org) to run the robots. Now we want to sufficiently document this design decision to make collaborators understand why we are using this framework.

---

## :white_check_mark: &nbsp; Conditions of satisfaction

- [ ] Should include a summary, WHY (maybe Pro / Cons List to compare other Frameworks / solutions) and HOW we are using RPA-Framwork (MD -> Wiki)
- [ ] Should include how we envision the framework being extended in the future (programming in Python / different layers between the framework & the Python code being executed).
- [ ] Should include a short feedback on how well you can work with this framework
- [ ] The required information for the presentation should be easily accessible from the wiki document

### PR DESCRIPTION
**Pull request by kej-jay**
fixes #88 

I wrote some documentation on why we decided to use the RPA framework

## Checklist before merge

**Developer's responsibilities**
* [X] **Assign** one or two developers
* [ ] **Change code** if reviewer(s) has/have requested it
* [X] updated the **documentation**

### CODE REVIEW

File: wiki/Why-RPA-framework.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Already offered functionality by the RPAf
+The RPAf already offeres a large suite of supported functionalities and softwares, of which we want to list some here:
+* SAP
+* Salesforce
+* MS Office
+* Email
+* Pdf
+* Image
+* HTTP request functionality
```
Comment (SanJSp) [inline wiki/Why-RPA-framework.md:12]: * Browser

File: wiki/Why-RPA-framework.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Because RPAf allows for easy creation of individual modules and libraries, we could be able to inject python code into the RPAf at runtime.
+This could be achieved by an additional parsing step, which would take the user provided code, would create a python file/library out of it and would at the same time automatically generate the keywords for use in the RPAf bot, once the custom library has been implemented and imported.
+
+# Alternatives 
+The RPAf follows a unique approach in being open-source, extensible and usable for a plethora of use-cases.
+While many other platformas allow for export of the there created bots through XML, they do not offer the documentation in such detail, as RPAf is doing.
+Because the goal of this project was to create an open source platform, we took into consideration options we had from teh open source spectrum of existing solutions:
+* TagUI: This RPA software allows for easy bot creation using teh keywords 'type' and 'click'. This approach is really easy, but would require a graphically oriented interface to display the user which options are available, which would not be optimal of use with our approach of bot-creation through a modeler.
```
Comment (SanJSp) [inline wiki/Why-RPA-framework.md:38]: Typo: the

File: wiki/Why-RPA-framework.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* OpenRPA: Is more focussed on the direction of having people use the internal design tool to create bots. Additionally the documentation is very barebones, which might be caused by the fact that the company standing behind this project also offers commercial support. Overall, it seemed not as easy to work with this project.
+* TaskT: When first inspecting this project, it seems like a very similar approach as the RPAf, but at the second look, it becomes clear, that the keywords used and displayed in the editor on TaskT are not only keywords, but only representations for the user to click to reveal the real underlying information. That fact, as well as the ineffective performance while testing caused us to decide against this project to use. 
+* UI.Vision: Was initially created as a browser extension, but since has been extended to also allow automation on the desktop. We ecplicitly want to stress teh fact, that it was created with web automation and scraping in mind and tehrefore works really well for this. For automating desktop applications we ran into problems when trying out this project.
```
Comment (SanJSp) [inline wiki/Why-RPA-framework.md:41]: typo: the

Comment (SanJSp) [reply wiki/Why-RPA-framework.md:41]: typo: therefore

Comment (SanJSp) [reply wiki/Why-RPA-framework.md:41]: typo: explicitly

**Review by SanJSp — CHANGES_REQUESTED** [review]
content great! Typos should be fixed.

**Review by lukashueller — APPROVED** [review]
Nice, great documentation. It will take a little effort to write out the main points for the presentation, but apart from that I can't think of any other points that need to be included in the documentation.

**Review by SanJSp — APPROVED** [review]

**Pull request merged**